priv/repo/migrations/20200804022935_create_user_settings.exs

```diff
@@ -0,0 +1,16 @@
+defmodule ChatApi.Repo.Migrations.CreateUserSettings do
+  use Ecto.Migration
+
+  def change do
+    create table(:user_settings, primary_key: false) do
+      add(:id, :binary_id, primary_key: true)
+      add(:email_alert_on_new_message, :boolean, default: false, null: false)
+
+      add(:user_id, references(:users, type: :integer), null: false)
```

Comment:
maybe it is better to use `:bigint` for fk

Reply:
yeah... we currently just use the default fields on the `users` table that pow (https://github.com/danschultzer/pow) gives us, but i'd love to use uuid's instead of integers here